autoscale/src/main/java/com/sequenceiq/periscope/controller/AlertController.java

```diff
@@ -229,14 +224,34 @@ private LoadAlertResponse createLoadAlertResponse(LoadAlert loadAlert) {
         return loadAlertResponseConverter.convert(loadAlert);
     }
 
-    private void validateTimeAlert(Long clusterId, Optional<Long> alertId, TimeAlertRequest json) throws ParseException {
-        validateAlertForUpdate(clusterId, alertId, AlertType.TIME);
-        dateService.validateTimeZone(json.getTimeZone());
-        dateService.getCronExpression(json.getCron());
+    private void validateTimeAlert(Long clusterId, Optional<Long> alertId, TimeAlertRequest json) {
+        alertId.ifPresent(alert -> validateAlertForUpdate(clusterId, alert, AlertType.TIME));
+        try {
+            dateService.validateTimeZone(json.getTimeZone());
+            dateService.getCronExpression(json.getCron());
+        } catch (ParseException parseException) {
+            throw new BadRequestException(parseException.getMessage(), parseException);
+        }
     }
 
-    private void validateLoadAlert(Long clusterId, Optional<Long> alertId, LoadAlertRequest json) throws ParseException {
-        validateAlertForUpdate(clusterId, alertId, AlertType.LOAD);
+    private void validateLoadAlert(Long clusterId, Optional<Long> alertId, LoadAlertRequest json) {
+        Cluster cluster = clusterService.findById(clusterId);
+        alertId.ifPresentOrElse(
+                updateAlert -> {
+                    validateAlertForUpdate(clusterId, updateAlert, AlertType.LOAD);
+                }, () -> {
+                    if (!cluster.getTunnel().useClusterProxy()) {
```

Comment:
This looks good based on the assumption that we're going with w.r.t all clusters using the ClusterProxy. Will have to include this in the documentation eventually.